cmdstanpy/utils/command.py

```diff
@@ -47,6 +48,7 @@ def do_command(
                 stderr=subprocess.STDOUT,  # avoid buffer overflow
                 env=os.environ,
                 universal_newlines=True,
+                encoding=locale.getdefaultlocale()[1],
```

Comment:
It's too bad `encoding='locale'` and `locale.getencoding()` are only available on newer versions. I think `locale.getpreferredencoding(False)` is the best option for older versions